src/Console/InputDefinition.php

```diff
@@ -23,6 +23,7 @@ public function __construct()
 
         $this->addOption(new InputOption('grep', 'g', InputOption::VALUE_REQUIRED, 'Run tests matching <pattern> <comment>(default: *.spec.php)</comment>'));
         $this->addOption(new InputOption('no-colors', 'C', InputOption::VALUE_NONE, 'Disable output colors'));
+        $this->addOption(new InputOption('--force-colors', '-f', InputOption::VALUE_NONE, 'Force output colors'));
```

Comment:
Sounds reasonable, I'll remove the shortcut.

Just a side question, the `option()` method in the very same class is doing the same thing with all these lines in the constructor, but they seems rarely used, I'm curious.